log4j-1.2-api/src/test/java/org/apache/log4j/config/PropertiesConfigurationTest.java

```diff
@@ -326,4 +326,9 @@ public void testEnhancedRollingFileAppender() throws Exception {
         }
     }
 
+    @Override
```

Comment:
Should I add test in `Log4j1ConfigurationFactoryTest`, too?
I don't understand the difference of `PropertiesConfigurationTest` and it.

Reply:
`Log4j1ConfigurationFactory` is an old factory that does not use builders. No need to add it there.